### DISCUSSION TITLE
issue with running on betterdiscord

### DISCUSSION THREAD

**Discussion by feralscales** [DaBluLite/DiscordColorways#2] [Q&A]
im using colorways on betterdiscord for the virtual boy theme. installing and running the colorways plugin works just fine, but, when i close discord and reopen it, i get an addon error saying colorways "could not be compiled". im not too sure what that means. pls help
![Screenshot 2023-12-27 130450](https://github.com/DaBluLite/DiscordColorways/assets/155011308/0408588e-4a4e-40c5-b99d-25e430c3d7f4)

**Comment by DaBluLite** [comment]
The plugin hasn't been fixed on betterdiscord since discord did the last batch of changes. I will fix it once I have time

**Comment by DaBluLite** [comment]
Forgot to close this, the BD plugin should work fine (for the most part)
